viite-UI/src/view/FormCommon.js

```diff
@@ -80,16 +81,30 @@
           }
       };
 
-    var roadTypeDropdown = function() {
-      var roadTypeDefaultValueToShow = '--';
-      var roadTypeDefaultValue = RoadType.Empty.value;
-      return '<select class="' + prefix + 'form-control" id="roadTypeDropdown" size = "1" style="width: auto !important; display: inline">' +
-        '<option value = "' + roadTypeDefaultValue + '">' + roadTypeDefaultValueToShow + '</option>' +
-        '<option value = "1">1 Maantie</option>' +
-        '<option value = "2">2 Lauttaväylä maantiellä</option>' +
-        '<option value = "3">3 Kunnan katuosuus</option>' +
-        '<option value = "4">4 Maantien työmaa</option>' +
-        '<option value = "5">5 Yksityistie</option>' +
+    var roadTypeLabel = function(roadType){
+      var roadTypeInfo = _.find(LinkValues.RoadTypeDisplayValues, function (obj) {
+        return obj.value === roadType;
+      });
+      return roadTypeInfo;
+    };
+    var roadTypeDropdown = function(roadTypeDefaultValue) {
+      var roadTypeDefaultValueToShow = '';
+      if (roadTypeDefaultValue === '') {
+        roadTypeDefaultValue = RoadType.Empty.value;
+        roadTypeDefaultValueToShow = '--';
+      } else {
+        var roadTypeText = roadTypeLabel(roadTypeDefaultValue);
+        roadTypeDefaultValueToShow = roadTypeText.description;
+
+      }
+
+      return '<select class="'+prefix+'form-control" id="roadTypeDropdown" size = "1" style="width: auto !important; display: inline">' +
+        '<option value = "' + roadTypeDefaultValue+ '" selected hidden >' +roadTypeDefaultValueToShow+'</option>' +
```

Comment:
and display directly the expected display text ..  `ected hidden >' +roadTypeLabel(roadTypeDefaultValue)+'</option>' +`